bedrock/base/templates/403_csrf.html

```diff
@@ -0,0 +1,41 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+ <!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <title>CSRF mismatch detected.</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style type="text/css">
```

Comment:
**Nit:** can we try and keep inline CSS out of templates (for you future CSP hopes and dreams)?

Reply:
Done!